engine/apps/slack/scenarios/step_mixins.py

```diff
@@ -1,72 +1,162 @@
+import json
 import logging
-from abc import ABC, abstractmethod
 
+from apps.alerts.models import AlertGroup
 from apps.api.permissions import user_is_authorized
+from apps.slack.models import SlackMessage, SlackTeamIdentity
 
 logger = logging.getLogger(__name__)
 
 
-class AccessControl(ABC):
+class AlertGroupActionsMixin:
+    """
+    Mixin for alert group actions (ack, resolve, etc.). Intended to be used as a mixin along with ScenarioStep.
+    """
+
     REQUIRED_PERMISSIONS = []
-    ACTION_VERBOSE = ""
```

Comment:
`ACTION_VERBOSE` is not needed anymore, it was used to send messages to AG thread with information about interaction attempt. Now it's a simple warning popup in case of authorization failure.